shell/models/fleet.cattle.io.gitrepo.js

```diff
@@ -20,6 +20,39 @@ function quacksLikeAHash(str) {
   return false;
 }
 
+function clusterIdFromLabels(labels) {
+  const clusterNamespace = labels?.[FLEET_ANNOTATIONS.CLUSTER_NAMESPACE];
+  const clusterName = labels?.[FLEET_ANNOTATIONS.CLUSTER];
+
+  return `${ clusterNamespace }/${ clusterName }`;
+}
+
+// bundleDeploymentResources extracts the list of resources deployed by a BundleDeployment
+function bundleDeploymentResources(status) {
+  // Use a map to avoid `find` over and over again
+  const resourceKey = (r) => `${ r.kind }/${ r.namespace }/${ r.name }`;
+
+  // status.resources includes of resources that were deployed by Fleet *and still exist in the cluster*
+  const resources = new Map((status?.resources || []).map((r) => [
+    resourceKey(r), Object.assign({ state: STATES_ENUM.READY }, r)]));
+
+  const modified = [];
+
+  for (const r of status?.modifiedStatus || []) {
```

Comment:
i couldn't see a modifiedStatus in a fresh 2.9.3 instance. is it new or situational?